benchpress/tket_gym/manipulate/test_manipulate.py

```diff
@@ -94,3 +94,26 @@ def result():
             return cu.circuit
 
         assert result
+
+    def test_random_clifford_decompose(self, benchmark):
+        """Decompose a random clifford into
+        basis [rz, sx, x, cz]
+        """
+        seqpass = SequencePass(
```

Comment:
What about this pass: https://tket.quantinuum.com/api-docs/transform.html#pytket.transform.Transform.OptimiseCliffords

Reply:
Added Tket Clifford optimization transformation